helm-chart/ray-cluster/values.yaml

```diff
@@ -100,6 +102,60 @@ worker:
     - mountPath: /tmp/ray
       name: log-volume
 
+extraWorkers:
+  generic:
+    # Disabled by default
+    disabled: true
```

Comment:
The double-negative `disabled : False` leads to some cognitive friction, so it would be better to have an `enabled` field.

Reply:
@DmitriGekhtman I was thinking that if I add a enabled: true, then it will not be backward compatible. Users that may forget to add this flag, they will generate a manifest that has a missing workergroup.

I am up for switching it to enabled: true

what do you think?

Reply:
Oh, I see, never mind -- `disabled` is fine.

Typically one would either do 
`disabled: true` OR
just omit the field entirely -- that's equivalent to `disabled: false`, but no one needs to include that.

Reply:
Never mind! :)